### PR TITLE
KOA-4805 Added missing raw text token aliases

### PR DESCRIPTION
**Pull request by marianeum**
Some of the raw token are missing for the new text styles, which causes issues with how the tokens are set up on Android. Move the tokens to common, as they're shared and add the missing tokens for the new text styles.

Remember to include the following changes:

- [x] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack-foundations/blob/main/CHANGELOG_FORMAT.md))
- [x] `README.md`
- [x] Tests
- [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: packages/bpk-foundations-android/src/base/typography.json [hunk 1]
```diff
@@ -56,76 +56,6 @@
       "type": "color",
       "category": "text-colors"
     },
-    "FONT_SIZE_CAPS": {
```
Comment (marianeum) [inline packages/bpk-foundations-android/src/base/typography.json:59]: moved these all to common

File: packages/bpk-foundations-android/tokens/base.android.xml [hunk 2]
```diff
@@ -105,6 +105,10 @@
   <property name="FONT_FAMILY" category="typesettings">"sans-serif"</property>
   <property name="FONT_FAMILY_EMPHASIZE" category="typesettings">"sans-serif-medium"</property>
   <property name="FONT_FAMILY_HEAVY" category="typesettings">"sans-serif-black"</property>
+  <property name="FONT_SIZE_5XL" category="typesettings">64sp</property>
```
Comment (marianeum) [inline packages/bpk-foundations-android/tokens/base.android.xml:108]: new tokens hooray

Comment (frugoman) [reply packages/bpk-foundations-android/tokens/base.android.xml:108]: :D

**Review by frugoman — APPROVED** [review]
Looking Good!!

**Pull request merged**